### PR TITLE
Check html activities output from running script

### PR DESCRIPTION
**Pull request by will-moore**
Adding test to check Activities html for script to catch bug at https://github.com/ome/omero-web/pull/61 that was found in manual testing.

Work in progress (having trouble running scripts locally)...

### CODE REVIEW

File: components/tools/OmeroWeb/test/integration/test_scripts.py [hunk 1]
```diff
@@ -49,11 +49,13 @@ def upload_script(self):
 
         script = """
 import omero
-from omero.rtypes import rstring, rint, wrap
+from omero.rtypes import rstring, rint, wrap, robject
```
Comment (jburel) [inline components/tools/OmeroWeb/test/integration/test_scripts.py]: maybe time to order alphabetically the import

Comment (joshmoore) [reply components/tools/OmeroWeb/test/integration/test_scripts.py]: adding

```
[flake8]
import-order-style = smarkets
```

to `.flake8` would be one method to enforce the imports. see https://pypi.org/project/flake8-import-order/ for more info.

Comment (will-moore) [reply components/tools/OmeroWeb/test/integration/test_scripts.py]: I'm not sure that flake8 would catch these imports since they are in a script string.

**Comment by will-moore** [comment]
This test now failing as expected with the fix https://github.com/ome/omero-web/pull/61 excluded due to travis

https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/53/testReport/OmeroWeb.test.integration.test_scripts/TestScripts/test_script_inputs_outputs_inputs0_/

**Pull request merged**